doc/classes/int.xml

```diff
@@ -380,27 +376,23 @@
 			<return type="int" />
 			<param index="0" name="right" type="int" />
 			<description>
-				Returns the result of bitwise [code]OR[/code] operation for two integers.
+				Performs the bitwise [code]OR[/code] operation.
 				[codeblock]
-				print(2 | 4) # 6
-				print(1 | 3) # 3
+				print(0b1100 | 0b1010) # Prints 14 (binary 1110)
 				[/codeblock]
-				It's useful to store binary flags in a variable.
+				This is useful for storing binary flags in a variable.
 				[codeblock]
 				var flags = 0
-				# Turn first and third bit on.
-				flags |= 1
-				flags |= 4
+				flags |= 0b101 # Turn the first and third bits on.
 				[/codeblock]
 			</description>
 		</operator>
 		<operator name="operator ~">
 			<return type="int" />
 			<description>
-				Returns the result of bitwise [code]NOT[/code] operation for the integer. It's effectively equal to [code]-int + 1[/code].
+				Performs the bitwise [code]NOT[/code] operation on the [int]. Due to [url=https://en.wikipedia.org/wiki/Two%27s_complement/]2's complement[/url], it's effectively equal to [code]-int - 1[/code].
 				[codeblock]
-				print(~4) # -3
-				print(~7) # -6
+				print(~4) # Prints -5
```

Comment:
If you are going to incorporate changes from https://github.com/godotengine/godot/pull/75325, I'd prefer their version (sans the new paragraph). And it would be worth adding @umarcor as a co-author of the PR.

Reply:
Doned. Since there's a link to Two's complement wikipedia page, I didn't add the binary representations; I just think it's counterintuitive to trim the 1's.